code/game/machinery/doors/airlock_control.dm

```diff
@@ -126,22 +126,7 @@
 	var/pressure
 
 /obj/machinery/airlock_sensor/buildable
-	uncreated_component_parts = list(
-		/obj/item/stock_parts/power/apc = 1
-	)
-
-/obj/machinery/airlock_sensor/Initialize(mapload, d, populate_parts)
-	. = ..()
-	update_icon()
-
-/obj/machinery/airlock_sensor/examine(mob/user, distance, infix, suffix)
```

Comment:
Was this examine text replaced somewhere?

Reply:
Probably something that slipped during the merge..